### ISSUE TITLE
Write example in Elixir

### ISSUE THREAD

**Issue by mickel8** [mickel8/quic_dist#18]

**Comment by sleipnir** [comment]
Hi @mickel8, has there been any progress here?
I'm interested in using quic_dist in a large project. More details about the project can be found here https://github.com/eigr/spawn/issues/90

We have a [discord channel](https://discord.gg/C8gpWJkJ) if you're interested in joining the discussions. We would love to try out using quic_dist for our purposes.

**Comment by mickel8** [comment]
Hi @sleipnir 

glad to hear that you are interested in using quic_dist.

Unfortunately, I have so much other work at the moment but PRs are very welcomed. I can also assist you in a free time :slightly_smiling_face:

**Comment by sleipnir** [comment]
> Hi @sleipnir
> 
> glad to hear that you are interested in using quic_dist.
> 
> Unfortunately, I have so much other work at the moment but PRs are very welcomed. I can also assist you in a free time slightly_smiling_face

Cool. Do you think the project is usable in production?

**Comment by mickel8** [comment]
I remember there was at least one bug i.e. sometimes you might not get connected?  https://github.com/mickel8/quic_dist/issues/20

**Comment by mickel8** [comment]
And you probably also need this https://github.com/mickel8/quic_dist/issues/21

**Comment by sleipnir** [comment]
Ok. Thanks!

**Comment by heri16** [comment]
This issue is still open?

**Comment by mickel8** [comment]
@heri16 Yes, unfortunately I am focused on other things at the moment but any PRs are welcomed
